actors/builtin/miner/miner_state.go

```diff
@@ -216,23 +209,17 @@ func (st *State) GetStorageWeightDescForSector(store adt.Store, sectorNo abi.Sec
 	return asStorageWeightDesc(st.Info.SectorSize, sectorInfo), nil
 }
 
+func (st *State) inChallengeWindow(rt Runtime) bool {
```

Comment:
Help us clean up debt and move this function (with rt dependency) to the actor itself.

Also this should check now < start + window

Reply:
> Help us clean up debt and move this function (with rt dependency) to the actor itself.

Ack

> Also this should check now < start + window

Nope, doesnt need to. Once you either submit your proof, or are slashed for being late, ProvingPeriodStart moves forward, meaning you'll never actually be in a situation where rt.CurrEpoch > ProvingPeriodStart+window

Reply:
Ok. Maybe comment that so future devs aren't confused like me.